runtime/ftplugin/lua.lua

```diff
@@ -1,9 +1,10 @@
 -- use treesitter over syntax
 vim.treesitter.start()
 
+vim.bo.includeexpr = 'v:lua.require"vim._ftplugin.lua".includeexpr()'
```

Comment:
For future reference: if/when options support function refs, then we could avoid the need for `vim._ftplugin`, like this:

```lua
vim.bo.includeexpr = function()
  ...
end
```